core/spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binding/BindingService.java

```diff
@@ -404,9 +405,8 @@ private void validate(Object properties) {
 	}
 
 	private void scheduleTask(Runnable task) {
-		Date d = new Date(System.currentTimeMillis()
-				+ this.bindingServiceProperties.getBindingRetryInterval() * 1_000);
-		this.taskScheduler.schedule(task, d.toInstant());
+		this.taskScheduler.schedule(task, Instant.ofEpochMilli(System.currentTimeMillis()
```

Comment:
**[UNRELATED]** I am expecting this to start failing on the CI build very soon

Reply:
@onobc This is already addressed yesterday by @olegz as CI started to fail.

Reply:
Ha! I figured that would (or had to) be the case. However, I saw it green in Jenkins and did not see a related commit so was a bit confused. Cool. This should go away during rebase (or I will pull this out and repush). Thanks for heads up.